demisto_sdk/commands/content_graph/interface/neo4j/queries/relationships.py

```diff
@@ -109,6 +109,41 @@ def build_uses_relationships_query(
 RETURN count(r) AS relationships_merged"""
 
 
+def update_alert_to_incident_relationships():
+    return f"""
+// Updated USES relationships between nodes when the source contains "alert" in the id,
+// This query changed the target node to content item that is in repository and its name is similar
+// to the old target with "incident" instead "alert" in the name.
+
+// Get USES relationship when target node contains "alert" and target node not in repository
+// and source not is in marketplacev2 and not xsoar
+MATCH (source)-[r:{RelationshipType.USES}]->(target)
+WHERE toLower(target.object_id) CONTAINS 'alert' AND target.not_in_repository
+   AND 'marketplacev2' IN source.marketplaces
+   AND NOT 'xsoar' IN source.marketplaces
+WITH target, source, r
+
+// get nodes with "incident" instead "alert" in the object_id that is in repository with the same
+// content type and its in marketplacev2 and xsoar
+MATCH (target_incident)
+WHERE toLower(target_incident.object_id) = REPLACE(toLower(target.object_id), 'alert', 'incident')
+    AND NOT target_incident.not_in_repository
+    AND target.content_type in labels(target_incident)
+    AND 'marketplacev2' IN target_incident.marketplaces
+    AND 'xsoar' IN target_incident.marketplaces
+WITH target_incident,target, source, r
+
+
+// create a relationship with the new incident node
+CREATE (source)-[r_new:{RelationshipType.USES}{{mandatorily: r.mandatorily}}]->(target_incident)
+
+
+// delete the old target node and old relationship
+DELETE r
```

Comment:
Isn't deleting the target enough?

Reply:
No, it failed and ask to delete the relationship before